.github/workflows/CI.yaml

```diff
@@ -239,7 +239,10 @@ jobs:
       - name: "Test: Process node runtime testsuite"
         run: ./scripts/gear.sh test rtest
 
-      # TODO: make also rtest run for node without lazy-pages feature (issue #1023)
+      - name: "Test: runtime spec tests no lazy pages"
+        run: |
+          cargo build -p gear-node --release --no-default-features --features=gear-native --out-dir target-no-lazy -Z unstable-options
```

Comment:
we can use `CARGO_TARGET_DIR` env var to avoid `-z unstable-options`

Reply:
`CARGO_TARGET_DIR` change will cause new build of all crates in this folder, while `--out-dir` use builded crates from current `CARGO_TARGET_DIR` and in out-dir will be only new binary.